google-cloud-spanner/lib/google/cloud/spanner/client.rb

```diff
@@ -52,11 +52,12 @@ class Client
         ##
         # @private Creates a new Spanner Client instance.
         def initialize project, instance_id, database_id, min: 2, max: 10,
-                       keepalive: 1500
+                       keepalive: 1500, write_ratio: 0.5, fail: false
```

Comment:
I think it would be helpful to somehow group the following options, which are all for session pool configuration: `min`, `max`, `keepalive`, `write_ratio`, and `fail`. Perhaps with a `pool_` prefix or else by grouping them in a hash under a `pool_options` or `pool_config` option?